src/lib/utils.js

```diff
@@ -1,13 +1,15 @@
 import reduce from 'lodash/reduce';
 import forEach from 'lodash/forEach';
 import find from 'lodash/find';
+import findIndex from 'lodash/findIndex';
```

Comment:
you're right but I think it has no extra cost here because we are already using `find` it depends of `findIndex` https://github.com/lodash/lodash/blob/4.17.11-npm/find.js#L2